src/gui/input/InputAPIAddWindow.cpp

```diff
@@ -114,6 +114,12 @@ InputAPIAddWindow::InputAPIAddWindow(wxWindow* parent, const wxPoint& position,
 	this->Bind(wxControllersRefreshed, &InputAPIAddWindow::on_controllers_refreshed, this);
 }
 
+InputAPIAddWindow::~InputAPIAddWindow()
+{
+	std::lock_guard lock{m_search_thread_data->mutex};
```

Comment:
`m_search_thread_data` can be null here if `on_add_button` wasn't called yet. This can be triggered when opening the window and just closing it again